### DISCUSSION TITLE
Add a 'Random' sort option to the list of albums?

### DISCUSSION THREAD

**Discussion by teranex** [music-assistant/support#2374] [Feature requests and ideas]
Would it be possible to add a Random sort option to the list of albums? Both Jellyfin and Symfonium have this option and I like it to just open the Album overview and see a random selection of my albums to select the next one I'll listen.

**Comment by SDoesburg** [comment]
I really miss this feature from Navidrome as well. It's great when I want something to listen to but I don't have anything particular in mind. If I don't see anything that I like I refresh the list to get a new selection at the top of the list. It helps me to listen to a lot of albums that I may not have listened to in a while.

**Reply by OzGav** [reply]
There is a random album playlist

**Reply by SDoesburg** [reply]
Yeah I have found that and it's almost what I was looking for. Is that new? I don't remember seeing it a couple weeks ago when I set everything up. I've noticed the frequent updates and I think the developers are doing a great job.

I'd like to have it give me a bunch of options to spark some inspiration, and the playlist serves them up one by one, which doesn't feel quite as nice as I'm used to from Navidrome.

**Reply by zecoj** [reply]
> There is a random album playlist

I don't think this has the same effect. A random list of albums gives you more choice instead of the random album playlist which is only one album essentially.

I too miss this feature coming from LMS. I rarely sort albums as a quick search would get me what I wanted. However to have a random view of what I have in my library is a cool way to re-discover forgotten albums.

**Reply by OzGav** [reply]
> a cool way to re-discover forgotten albums.

There is also the `last played` or `play count` options which should also show you those forgotten albums.

**Reply by sebcoh** [reply]
I would also love a random sort option. I am coming from LMS too and it's a fantastic way to find albums that fit my current mood without bias that scrolling through sorted playlists would introduce.
The random album playlist is nice but very tedious compared to a random album list.

**Comment by sunjaxx** [comment]
I would add a random sorting option to "Tracks" (and "Artists") as well!

**Comment by brianjmccauley** [comment]
I would love a random sort option in Albums (Track and Artists as well). Great way to stumble on music in my collection I haven't heard in a while.

**Comment by dentedbread** [comment]
I have switched from Emby to Music Assistant and this "Random album sort order" is one feature I miss very much.  Please consider implementing it.

**Comment by teranex** [comment]
I found kind of a work around for now: I added the "500 Random tracks" playlist to my shortcuts. When you open that it shows you 500 random tracks. I scroll through it to find an album I want to play. Then, instead of clicking on the play button, I click on the track title which brings me to the detail page of that track. There I find the 'appears on', and I can start the full album.

**Reply by zecoj** [reply]
Appreciate you sharing the work around but what a shame that a seemingly trivial improvement has been forgotten for over 2 years. I've been refreshing MA's home page to get the Random Albums row/ribbon but that's limited to 10 albums.

**Reply by teranex** [reply]
One other way, if you use Android, is to use the MassDroid (https://github.com/sfortis/massdroid_native) app. That does have a random sort option for albums.

**Reply by OzGav** [reply]
@zecoj this isn’t “forgotten”. This has 11 votes after 2 years so is not high priority. However, a community member has opened a PR for this so will be in the beta in coming weeks.
